File: src/model/StatusOfBillEnum.ts
export enum StatusOfBillEnum {
  OPENED = 'Aberto',
  PARTIALLY = 'Parcialmento pago',
  CANCELLED = 'Cancelado',
  SETTLED = 'Pago/Quitado',
  RENEGOTIATED = 'Renegociado',
  REFUND = 'Devolvido'
}

export const StatusOfBillList: {
    key: string;
    value: string;
}[] = Object.entries(StatusOfBillEnum).map(([key, value]) => ({ key, value }))

export type StatusOfBillType = keyof typeof StatusOfBillEnum

export const StatusOfBillEntries = Object.entries(StatusOfBillEnum)

File: src/main/java/br/com/api/enums/StatusOfBillEnum.java
package br.com.api.enums;

import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
@Getter
public enum StatusOfBillEnum {

	CREATED("label.created"),
	OPENED("label.opened"),
	PARTIALLY("label.partially"),
	CANCELLED("label.cancelled"),
	SETTLED("label.settled"),
	RENEGOTIATED("label.renegotiated"),
	REFUND("label.refund");

	private final String i18n;
}
